### PR TITLE
Fix #21: Add build scripts to publish ESM/Amd/CJS using microbundle

### LINKED ISSUE #21: Add build scripts to publish ESM/Amd/CJS etc

### PR DESCRIPTION
**Pull request by kokikathir**
Signed-off-by: Kokilavani Kathiresan <kathiresankokilavani@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,16 @@
   "version": "1.2.0",
   "description": "A parser for the Java/Maven pom.xml files.",
   "license": "MIT",
-  "main": "lib/",
+  "source": "lib/index.js",
+  "main": "dist/index.js",
+  "umd:main": "dist/index.js",
```
Comment (nsinghal12) [inline package.json:8]: Please use `index.umd.js` as the filename.

File: package.json [hunk 2]
```diff
@@ -3,7 +3,16 @@
...
+  "module": "dist/index.mjs",
```
Comment (nsinghal12) [inline package.json:9]: Use `index.esm.js` as the filename.

File: package.json [hunk 3]
```diff
@@ -3,7 +3,16 @@
...
+  "types": "dist/index.d.ts",
+  "exports": {
+    "types": "./dist/index.d.ts",
+    "require": "./dist/index.js",
+    "default": "./dist/index.modern.mjs"
```
Comment (nsinghal12) [inline package.json:14]: Use `index.modern.js` as the filename.

**Review by nsinghal12 — CHANGES_REQUESTED** [review]
Thanks for your PR. Requesting changes.

File: package.json (hunk 1 above)
Comment (kokikathir) [reply package.json]: done

File: package.json (hunk 2 above)
Comment (kokikathir) [reply package.json]: done

File: package.json (hunk 3 above)
Comment (kokikathir) [reply package.json]: done

**Review by nsinghal12 — APPROVED** [review]
Looks good to me. Thanks for contributing.

**Pull request merged**